x-pack/plugins/security/server/audit/audit_service.ts

```diff
@@ -154,9 +158,13 @@ export class AuditService {
       }
     };
 
+    const isLoggingEnabled = () => {
+      return this.logger.isLevelEnabled('info');
+    };
```

Comment:
So, technically there's a `enabled` flag passed though the whole service's `setup` method. However I felt more confident directly using the `isLevelEnabled` API, as it felt safer if we were one day going to support hot reloading of audit logging.

Please tell me if we'd prefer the `enabled` flag